.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: Build
+
+on:
+#  push:
+#    branches:
+#      - main
+  pull_request:
+  workflow_dispatch:
+
+# we might need this to allow queued workflows to interrupt previous runs
+#concurrency:
+#  group: ${{ github.workflow }}-${{ github.event.number || github.ref }}
+#  cancel-in-progress: true
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: corretto
+          cache: sbt
+          java-version: 8
+
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: eu-west-1
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+
+      - name: Clean, compile and test
+        run: sbt clean compile test assembly
```

Comment:
I think the assembly command worked for me here because it generated the .zip we ultimately upload to the riff-raff-artifacts bucket (in the subsequent step)